roles/forklift/tasks/main.yml

```diff
@@ -9,16 +9,34 @@
 
 - name: 'Check Forklift state'
   fail:
-    msg: 'The forklift_state variable is undefined. Please use pass "-e "forklift_state=up"" to spin up the boxes or "-e "forklift_state=destroy"" to destroy the boxes'
-  when: forklift_state is undefined
+    msg: 'The invalid value for forklift_state variable. Please use pass "-e "forklift_state=up"" to spin up the boxes or "-e "forklift_state=destroy"" to destroy the boxes'
```

Comment:
The wording here in the first sentence is a little hard to parse.

Reply:
Not sure what I was thinking - probably not all.